slackbot_using_azure_openai/Ballerina.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+org = "ai_samples"
+name = "slackbot_using_azure_openai"
+version = "0.1.0"
+distribution = "2201.4.0"
+
+[build-options]
+observabilityIncluded = true
+
+[[dependency]]
```

Comment:
We need to remove local dependency once published.

Reply:
Ack. Will remove once the connector is published. Or shall we remove it now itself?

Reply:
The local dependency is removed in [9fee54c](https://github.com/ballerina-guides/ai-samples/pull/64/commits/9fee54cbc7a694253e8bfb098ff851e80616c750)